### ISSUE TITLE
Why limit to 13 digits number

### ISSUE THREAD

**Issue by enigma158an201** [GeorgeYT9769/cardabase-app#3]
I was happy to find such an adfree application, but first card i want to create is 14 digits, would it be possible to allow more than 13?

**Comment by GeorgeYT9769** [comment]
Yes, I am working on it with other updates as exporting and importing cards...

But right now it's only possible 13 and 8 in the next update, since package the app uses only supports ean-13 and ean-8 barcodes.

If you find anything helpful that can be added in the app, let me know!

Also thanks for using the app 🥹

**Comment by GeorgeYT9769** [comment]
Also, those packages that allow me to generate barcodes have to be FOSS for app to be accepted for F-Droid. So even if I find compatible package that allows generating custom barcodes, it has to be FOSS.

Before the app was released on F-Droid, it used more popular package for generating barcodes, but sadly, ot wasn't FOSS. So I had to rework it and stick with current solution. At this point I am happy that I made something work 🤣.

But don't worry, I work on it 💪.

**Comment by GeorgeYT9769** [comment]
So update 1.2.0 came out, with support for QR codes, CODE 128, 39 and 93 and other types I don't remember lol. It's live now on GitHub, but to get on F-Droid you will have to wait a few days.
